pkgs/top-level/all-packages.nix

```diff
@@ -16464,92 +16464,7 @@ let
 }; # self_ =
 
 
-  ### Deprecated aliases - for backward compatibility
```

Comment:
I think we do want to keep this comment or similar in this file. It makes it easier for new-comers to find the right file.